src/glossary.rst

```diff
@@ -3521,7 +3531,9 @@ lifetime elision
 ^^^^^^^^^^^^^^^^
 
 :dp:`fls_dq5wkd61ry3l`
-:dt:`Lifetime elision` is a set of relaxations on the use of :t:`[lifetime]s`.
+:dt:`Lifetime elision` is a set of rules that automatically insert :t:`[lifetime
+parameter]s` and/or :t:`[lifetime argument]s` when they are elided in the source
+code.
```

Comment:
```suggestion
:dt:`Lifetime elision` is a set of rules that automatically insert
:t:`[lifetime parameter]s` and/or :t:`[lifetime argument]s` when they are
elided in the source code.
```
Not splitting terms that consist of multiple words across multiple lines makes it easier to grep. I went through all the chapters and ensured that none of the terms are split.